### PR TITLE
Update to angular 1.5.11 and patternfly 3.18.1

### PR DESCRIPTION
**Pull request by dtaylor113**
Hi,
Updating this component to latest angular 1.5.x and patternfly 3.x.

- all unit tests pass
- manually ran through many test pages
- could not get e2e testing running on my system: "WebDriverError: unknown error: Chrome version must be >= 53.0.2785.0" I'm using Chrome on Linux v50.0.2661.86 :-(

Thanks,
- Dave

### CODE REVIEW

**Comment by benjaminapetersen** [comment]
@dtaylor113 you have an error in `package.json` (comma), might be why it doesnt run locally for you.  Travis couldn't finish the build.

**Comment by benjaminapetersen** [comment]
@dtaylor113 we should bump the version # in `bower.json` and `package.json` as well.  If the file path for patternfly is different for building the CSS, we may have to make a major version bump to signify breaking changes. Thx!

File: package.json [hunk 1]
```diff
@@ -44,5 +44,6 @@
     "karma-phantomjs-launcher": "^1.0.1",
     "karma-safari-launcher": "^1.0.0",
     "karma-spec-reporter": "0.0.26"
```
Comment (benjaminapetersen) [inline package.json]: comma here

**Comment by dtaylor113** [comment]
Hi @benjaminapetersen, 
Added the missing coma.  Please let me know if you also want me to bump version numbers.
Thanks,
- Dave

**Comment by benjaminapetersen** [comment]
Sounds good Dave, lets definitely bump the version #. I'd probably say `3.0.0` to indicate its breaking. Lets see if Travis will run your tests now.

**Comment by dtaylor113** [comment]
Ok, bumped to 3.0.0.
Thanks,
- Dave

**Comment by benjaminapetersen** [comment]
LGTM, thanks Dave!

**Comment by benjaminapetersen** [comment]
@dtaylor113 mind a squash? Then I can merge this.

**Comment by dtaylor113** [comment]
squashed :-)

**Pull request merged**